### ISSUE TITLE
Address checksum mismatch. 

### ISSUE THREAD

**Issue by Ishitori** [almindor/etherwall#12]
Hi,

I want to send some ETC to my Poloniex account. I paste account number to "To" field: 0xdeed1ac845ab58e7e5c9728918d93c38ca0f7a5e

When I do so I get the following warning: Address checksum mismatch. Use only properly checksumed addresses to prevent wrong sends.

I wonder, is it a bug in Etherwall, or my account is really kinda screwed and I should contact Poloniex. I am using the latest version 1.4.4.

Thanks.

**Comment by almindor** [comment]
Better safe than sorry :)

It's not a bug but Poloniex should update their software. If you look up the address at etherscan.io for example ( https://etherscan.io/address/0xdeed1ac845ab58e7e5c9728918d93c38ca0f7a5e ) you see how it's capitalized in their website?

That capitalization is a form of checksum as described by Vitalik at https://github.com/ethereum/EIPs/issues/55

It's not REQUIRED however all the quality clients use it (Mist, Etherwall etc.)
All lowercase basically means the address is not "checksum-ed" and could contain a typo easily.

So TLDR: your address is probably fine, but it doesn't contain the optional capitalization "checksum" that would prevent typo mistakes.

**Comment by almindor** [comment]
I'll add a FAQ entry so people don't get confused over this.

**Comment by Ishitori** [comment]
Got it. Thank you!
